client/go/auth0/auth0.go

```diff
@@ -166,13 +166,7 @@ func (a *Auth0) PrepareSystem(ctx context.Context) (*System, error) {
 
 		res, err := tr.Refresh(ctx, a.system)
 		if err != nil {
-			// ask and guide the user through the login process:
```

Comment:
Not entirely sure about this one. Automatically renewing the token makes it all a bit easier for the user. But then again, if there is an intermittent network issue that casued the refresh to fail, starting the entire flow again is not really needed.

What is the lifetime of a refresh token?